README.md

```diff
@@ -1,39 +1,87 @@
-## JetBrainsRunner
+![JetBrains Toolbox Logo](https://cdn.icon-icons.com/icons2/1381/PNG/64/jetbrainstoolbox_93803.png)
+<sub> [Icon from icon-icons.com](https://icon-icons.com/icon/jetbrains-toolbox/93803#64) </sub>
 
-This plugin allows you to launch your recent JetBrains projects from Krunner.  
-Furthermore the search suggestions and the application -> config file mapping can be customized.  
-Additionally it has file watchers implemented, these reload the configuration when
-new projects get added/removed.  
+# JetBrainsRunner
+This plugin allows you to launch your recent JetBrains projects from Krunner.
 
-New Feature:
-Now you can pin your projects by dragging the KRunner results to the taskbar or desktop.
+- The search suggestions and the application ⇒ config file mapping can be customized.
+- File watchers are implemented, these reload the configuration when new projects get added/removed.
+- You can pin your projects by dragging the KRunner results to the taskbar or desktop.
 
-To install the plugin you can visit out the release page(https://github.com/alex1701c/JetBrainsRunner/releases)
-or build from source as explained below.
+<details>
+<summary><b>Screenshots</b></summary>
```

Comment:
I am wondering if it is a good idea to all screenshots by default.

For the config dialog I agree, that might not be important when one gets an initial impression of what this plugins does. However I think providing a simple screenshot how the matches will be displayed helps the user. (That is how I felt when first scrolling through the available KRunner plugins on github a few years ago ;))